### ISSUE TITLE
Error in client.website.add_new_people_profile

### ISSUE THREAD

**Issue by catacs** [crisp-im/ruby-crisp-api#1]
I am trying to use the api method client.website.add_new_people_profile and i get this error:

```
    /gems/crisp-api-1.0.2/lib/crisp-api.rb:114:in `_do_request': invalid_data (RouteError)
        from /gems/crisp-api-1.0.2/lib/crisp-api.rb:65:in `post'
        from /gems/crisp-api-1.0.2/lib/resources/website.rb:189:in `add_new_people_profile'
        from crisp_test.rb:14:in `<main>'
```

it seems the method  `add_new_people_profile` is not using the data parameter:

```
def add_new_people_profile(website_id, data)
    return @parent.post(self._url_website(website_id, "/people/profile"))
end
```

it should be:

```
def add_new_people_profile(website_id, data)
    return @parent.post(self._url_website(website_id, "/people/profile"), data: data)
end
```

**Comment by valeriansaliou** [comment]
Thanks, issuing a fix now.

**Comment by valeriansaliou** [comment]
All fixed, you can install `v1.0.3` from Rubygems.
